IrisSoftware/main.py

```diff
@@ -50,28 +51,42 @@ def detectBlink(self, eyeCoords, blinkDuration) -> any:
         pass
 
     def moveMouse(self, screenX, screenY):
-        '''Move the mouse to the given screen coordinates, moving smoothly over multiple frames'''
+        """Move the mouse to the given screen coordinates, moving smoothly over multiple frames"""
         if self.state.lastCursorPos is None:
             pyautogui.moveTo(screenX, screenY)
             self.state.lastCursorPos = (screenX, screenY)
         else:
             # Smooth out the mouse movement to minimize jitter
-            x = self.state.lastCursorPos[0] + (screenX - self.state.lastCursorPos[0]) * 0.1
-            y = self.state.lastCursorPos[1] + (screenY - self.state.lastCursorPos[1]) * 0.1
+            x = (
+                self.state.lastCursorPos[0]
+                + (screenX - self.state.lastCursorPos[0]) * 0.1
+            )
+            y = (
+                self.state.lastCursorPos[1]
+                + (screenY - self.state.lastCursorPos[1]) * 0.1
+            )
 
             pyautogui.moveTo(x, y)
             self.state.lastCursorPos = (x, y)
 
     def safeComputeCoords(self, eyeCoords):
-        # return last cursor position if available when eyes aren't properly detected, if not return center screen 
+        # return last cursor position if available when eyes aren't properly detected, if not return center screen
         if len(eyeCoords) < 2:
             if self.state.lastCursorPos is not None:
                 return self.state.lastCursorPos
             res = list(self.camera.getResolution())
-            return tuple([resolution //2 for resolution in res])
-        
+            return tuple([resolution // 2 for resolution in res])
+
         return self.interpolator.computeScreenCoords(eyeCoords)
 
+    def changePupilModel(self, value: PupilModelOptions):
+        if value == PupilModelOptions.ACCURACY:
+            self.eyeDetector.detectionType = (
+                EyeDetection.DetectionType.FACE_EYE_CASCADE_BLOB
+            )
+        elif value == PupilModelOptions.SPEED:
+            self.eyeDetector.detectionType = EyeDetection.DetectionType.EYE_CASCADE_BLOB
+
```

Comment:
I did add a setter method in the EyeDetection class, so you could change this to use `self.eyeDetector.setDetectionType(...)`

Reply:
Done!